CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ set(Boost_USE_STATIC_RUNTIME OFF)
 
 
 find_package(Boost COMPONENTS system thread program_options filesystem timer REQUIRED)
-find_package(ISMRMRD REQUIRED)
+find_package(ISMRMRD REQUIRED 1.7.0)
```

Comment:
I think this should be:

```
find_package(ISMRMRD 1.7.0 REQUIRED)
```

Please verify. In the current form it does nothing to pin down the version, but it does the way I have written it above. Please edit and validate that it does indeed fail if the version is not met and passes if it is.

Reply:
oops. sorry. late in the night. Confirmed
```
  Could not find a configuration file for package "ISMRMRD" that is
  compatible with requested version "1.7.0".

  The following configuration files were considered but not accepted:

    /home/sirfuser/devel/install/lib/cmake/ISMRMRD/ISMRMRDConfig.cmake, version: 1.5.0
    /home/sirfuser/devel/buildVM/builds/ISMRMRD/build/ISMRMRDConfig.cmake, version: 1.5.0
    /home/sirfuser/devel/build/SIRF-SuperBuild-test/builds/ISMRMRD/build/ISMRMRDConfig.cmake, version: 1.4.2
```
(surprised CMake looked in so many places!)